menus/pause_menu.gd

```diff
@@ -1,11 +1,24 @@
 extends Control
 
+var game_paused : bool = false : get = _get_game_paused, set = _set_game_paused
 
-# Called when the node enters the scene tree for the first time.
 func _ready():
-	pass # Replace with function body.
+	hide()
 
+func _get_game_paused():
+	return game_paused
+	
+func _set_game_paused(new_state: bool):
+	game_paused = new_state
+	get_tree().paused = game_paused
+	show() if game_paused else hide()
+
+func _input(event : InputEvent):
+	if(event.is_action_pressed("ui_cancel")):
+		game_paused = !game_paused
+
+func _on_resume_button_pressed():
+	game_paused = false
 
 func _on_quit_button_pressed():
-	pass
-	
+	get_tree().change_scene_to_file("res://menus/menu.tscn")
```

Comment:
You need to unpause the game before change the scene: `game_paused = false` 😅

Reply:
yea... Kinda forgot about that ahah